frontend/app/services/GuardianContentService.scala

```diff
@@ -52,16 +52,29 @@ trait GuardianContentService extends GuardianContent {
     enumerator(Iteratee.consume()).flatMap(_.run)
   }
 
+  private def membersOnly: Future[Seq[Content]] = {
+      for {
+        response <- membersOnlyContentQuery(1)
+      } yield {
+        response.results
+      }
+  }
```

Comment:
You could do `membersOnlyContentQuery(1).map(_.results)` if you're getting just one page of content.